# Generated by Django 2.2.6 on 2019-11-06 14:56

from django.conf import settings
import django.core.validators
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('web_gc', '0012_auto_20191101_1639'),
    ]

    operations = [
        migrations.AddField(
            model_name='cadastrotalao',
            name='user',
            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='entregatalao',
            name='user',
            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AddField(
            model_name='entregavale',
            name='user',
            field=models.ForeignKey(default=None, on_delete=django.db.models.deletion.PROTECT, to=settings.AUTH_USER_MODEL),
        ),
        migrations.AlterField(
            model_name='talao',
            name='talao',
            field=models.IntegerField(unique=True, validators=[django.core.validators.MinValueValidator(100, 'Talão inválido'), django.core.validators.MaxValueValidator(999999, 'Talão inválido')]),
        ),
    ]
